src/utils/resumeAnalysis.ts

```
@@ -1,98 +1,745 @@
+      return s?.label || s?.skillName || s?.name || '';
+    })
+    .filter(Boolean)
+    .map((s) => s.toLowerCase().trim());
+}
+
+function getAllText(r: ParsedResume): string {
+  const parts: string[] = [];
+  r.personalData.forEach((p) => {
+    parts.push(p.firstName || '', p.lastName || '', p.email || '');
+  });
+  r.summaryData.forEach((s) => parts.push(s));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)